tools/minikube/scripts/start_pods.sh

```diff
@@ -27,6 +27,21 @@ if [[ -z "${PINAKES_CONTROLLER_VERIFY_SSL}" ]]; then
   exit 1
 fi
 
+# Set the environment variable for accessing insights service
+# export PINAKES_INSIGHTS_TRACKING_STATE=True|False
+# export PINAKES_INSIGHTS_URL=<<your insights url>>
+
+# Check if metrics collection is turned on
+PINAKES_INSIGHTS_TRACKING_STATE=${PINAKES_INSIGHTS_TRACKING_STATE:-False}
+PINAKES_INSIGHTS_URL=${PINAKES_INSIGHTS_URL:-}
+
+if [[ "${PINAKES_INSIGHTS_TRACKING_STATE}" = "True" ]]; then
```

Comment:
@hsong-rh Is this needed now because we have a default for PINAKES_INSIGHTS_URL a customer doesn't have to set it anymore unless we are doing dev/qe testing and we want to point to the ingress in staging or a different env. For production it will work out of the box.